composer/algorithms/no_op_model/no_op_model.py

```diff
@@ -59,10 +65,18 @@ def validate(self, batch: Batch) -> Tuple[Any, Any]:
 class NoOpModel(Algorithm):
 
     def match(self, event: Event, state: State) -> bool:
+        """Runs on Event.INIT"""
```

Comment:
For the algorithms, can you remove the docstrings on `match` and `apply`, and instead elaborate what happens in the class-level docstring? We've generally been going with the convention that inherited methods do not have docstrings, unless if they deviate from the parent class.